force-app/main/default/classes/SOQL.cls

```diff
@@ -142,16 +192,16 @@ public with sharing class SOQL {
      * @return A list of Contact LastName.
      */
     public static List<String> getContactNamesByAccount(Id accountId) {
-        // Create a list to hold the Contact LastNames
-
-        // Query the Contacts
-        // Loop through each Contact and add the LastName to the list
-        //for () {
-        //    
-        //}
-
-        // Return the list of Contact LastNames
-        return null; // Replace null with the variable you used to store the result
+        List<String> myListOfContactLastNames = new List<String>();
+        List<Contact> contactListFromAccounts = new List<Contact>([
```

Comment:
This can also get put directly into the for loop instead of creating a new list variable